flight/run_mission.py

```diff
@@ -5,28 +5,38 @@
 import logging
 
 from mavsdk import System
+from flight import intake_gps, landing
 
-from intake_gps import Waypoint, extract_gps
-from landing import manual_land
+# from intake_gps import Waypoint, extract_gps
 import argparse
+import sys
 
+# sys.path.append("./flight/intake_gps.py")
 
-async def run_mission(path: str = "flight/data/target_data.json") -> None:
+
+async def run_mission(competition: bool) -> None:
     """
     Uses data from a json file to retrieve a mission then runs it to get the drone above the target
     once the drone gets 225 feet above the ground the landing code is run which brings it down
     at progressively slower speeds precisely over the target till it lands the drone and shut down
 
     Parameters
     ----------
-    path : str
-        File path to the target data JSON file.
+    competition : bool
+        Decides if competition waypoints are used in the mission or not.
 
     Notes
     -----
     Drone will be shut off after this is run
     It can be run by python3 run_mission.py -file {json file path}
     """
+
+    waypoint_path: str
+    if competition:
+        waypoint_path = "flight/data/target_data.json"
+    else:
+        waypoint_path = "flight/data/golf_target.json"
+
```

Comment:
Feel like `waypoint_path` should be renamed. Unless I'm mistaken, you should only be getting one point from the json file, correct?